app/src/util/validation.ts

```diff
@@ -245,3 +245,40 @@ export const updatePasswordRequest = z.object({
 });
 
 export type UpdatePasswordRequest = z.infer<typeof updatePasswordRequest>;
+
+export const createOrganizationRequest = z.object({
+  name: z.string().max(255),
+  contactEmail: z.string().email().max(255),
```

Comment:
Do we need to enforce these limits of 255 chars? There could reasonably be longer input for some situations?